### DISCUSSION TITLE
Performance related mods

### DISCUSSION THREAD

**Discussion by dracoattila** [Sinytra/ForgifiedFabricAPI#35] [Q&A]
Whenever I tried to use a performance related mod, be it Forge based like OptiFine, Rubidium ... or Fabric based such as Sodium, Indium, always caused a crash, while the game was loading or in-game.

The in-game crash happened with a conflict of Rubidium (Forge) and Mystic Mounts (Fabric) error. But viewing as the 2 mods have different modloader I accepted that they won't work together.

Is there any similar mod that can be used instead of OptiFine or Sodium with Connector?

**Comment by Shiarbme** [comment]
I've been able to run with Embeddium just fine, so give that a go.
